src/main.rs

```diff
@@ -57,7 +58,24 @@ fn main() {
         .on_state_update(
             APPSTATES,
             AppState::MainMenu,
-            menu::button_interact.system(),
+            menu::button_interact.system().label("buttons"),
```

Comment:
I think it would be good to keep all const strings in a constant somewhere like `const BUTTON_LABEL = "buttons"`, so make sure that we don't have issues with a typo.